cli/src/commands/logs.ts

```diff
@@ -0,0 +1,39 @@
+import { Command, flags } from '@oclif/command';
+
+export default class Logs extends Command {
+  public static description = 'Show logs';
+
+  public static examples = [
+    '$ shift-cli logs',
+    '$ shift-cli logs --follow',
+    '$ shift-cli logs --since 2018-03-09T22:12:21.861Z',
+    '$ shift-cli logs --since 3d',
+    '$ shift-cli logs --since 13hours',
+    '$ shift-cli logs --since 9s*',
+    '$ shift-cli logs --since 2m --follow',
```

Comment:
The examples had descriptions in the spec.

Reply:
I think it looks nicer this way, also this is how examples are usually shown AFAIK
https://github.com/heroku/cli/blob/master/docs/logs.md#heroku-logs

Will replace with the text from the spec if still an issue

Reply:
Please conform to the spec.

Reply:
fixed